templates/include/glean/custom_distribution.1.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "type": "object",
+  "properties": {
+    "range": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "type": "number"
+      }
+    },
+    "bucket_count": {
+      "type": "integer"
+    },
+    "histogram_type": {
+      "type": "string",
+      "enum": [
+        "linear",
+        "exponential"
+      ]
+    },
+    "values": {
+      "type": "object",
+      "propertyNames": {
+        "pattern": "[0-9]+"
+      },
+      "additionalProperties": {
+        "type": "integer"
+      }
+    },
+    "underflow": {
+      "type": "integer"
+    },
+    "overflow": {
+      "type": "integer"
+    },
+    "sum": {
+      "type": "integer"
+    }
+  },
+  "required": [
```

Comment:
Shouldn't `histogram_type` be required, so we can always calculate what the buckets are?

Reply:
Good catch.